lib/list_versions.go

```diff
@@ -126,17 +109,71 @@ func GetTFURLBody(mirrorURL string) ([]string, error) {
 		os.Exit(1)
 	}
 
-	body, errBody := ioutil.ReadAll(resp.Body)
-	if errBody != nil {
-		log.Printf("[Error] : reading body: %v", errBody)
+	body := new(bytes.Buffer)
+	if _, err := io.Copy(body, resp.Body); err != nil {
+		log.Fatal(err)
+	}
+	if err := json.Unmarshal(body.Bytes(), &releases); err != nil {
+		log.Fatalf("%s: %s", err, body.String())
+	}
+	return releases, nil
+}
+
+func GetTFReleases(mirrorURL string, preRelease bool) ([]Release, error) {
+	// temp testing
+	queryParams := map[string]string{"limit": "20"}
```

Comment:
Would it make sense to put `20` into a variable just in case they change their mind and allow a different limit so that we change the var value closer to the header of this file rather than looking up this value somewhere in the middle of the file? 🤔

Reply:
Agreed, fixed.